### ISSUE TITLE
Service port naming issue

### ISSUE THREAD

**Issue by kollerdroid** [loft-sh/devspace#1472]
**What happened?**  

According to the [Component Chart documentation](https://devspace.sh/component-chart/docs/configuration/service#name) we can change a port name in a service.

I tried to set it:

```yaml
 #...
        service:
          ports:
            - port: ${GRPC_PORT}
              name: grpc
 #...
```
I get an error:

[fatal]  deployments[0].helm.componentChart: component values are incorrect: yaml: unmarshal errors:
  line 41: field name not found in type latest.ServicePortConfig

**What did you expect to happen instead?**  

It should set the port name to __*grpc*__ (instead of __*port-0*__)

**How can we reproduce the bug?** (as minimally and precisely as possible)  

I get this error everytime.

**Local Environment:**  
- DevSpace Version: 5.14.1
- Operating System: Windows 10 WSL2  (Ubuntu)
- Deployment method: Component Chart

**Kubernetes Cluster:**  
- Cloud Provider: on-prem
- Kubernetes Version: v1.19.6

**Anything else we need to know?**  

/kind bug

**Comment by FabianKramm** [comment]
@kollerdroid thanks for creating this issue! You are correct, this field is not included in the config schema which is why DevSpace fails. We add it for the next version v5.14.2

**Comment by FabianKramm** [comment]
@kollerdroid should be fixed with v5.14.2
